theme/js/main.js

```diff
@@ -1,3 +1,6 @@
-const {message} = require("./_module");
-
-console.log(message);
+// Mobile Menu
+const mobMenu = document.querySelector("#mobile-nav");
```

Comment:
Suggestion: We don't need to listen to multiple events inside  `#mobile-nav` element, so we can instead use selectbox, i.e.: 
```
const mobMenu =  document.querySelector("#mobile-nav select");
...
window.location.pathname = mobMenu.selectedOptions[0].value;
```